apps/api/package.json

```diff
@@ -51,6 +51,7 @@
     "nodemailer": "^6.9.4",
     "reflect-metadata": "^0.1.13",
     "rxjs": "^7.8.1",
+    "twilio": "^5.0.0-rc.1",
```

Comment:
Usually don't consider a release candidate, they will not be stable (pre release tags).

Okay for now but keep that in mind.